Mage/src/main/java/mage/MageIdentifier.java

```diff
@@ -19,7 +31,65 @@ public enum MageIdentifier {
     WishWatcher,
     GlimpseTheCosmosWatcher,
     SerraParagonWatcher,
-    OneWithTheMultiverseWatcher,
+    OneWithTheMultiverseWatcher("Without paying manacost"),
     JohannApprenticeSorcererWatcher,
-    KaghaShadowArchdruidWatcher
+    KaghaShadowArchdruidWatcher,
+    CourtOfLocthwainWatcher("Without paying manacost"),
```

Comment:
not part of this PR (will be used in the next one), but presumably still safe to include here anyway

Reply:
Yes, extra enum field not used is no harm.
Was easier to test the Court of Lochtwain code in addition to this branch. I can clear it if you mind it being there.